source/views/building-hours/report/editor.js

```diff
@@ -191,32 +187,47 @@ class DatePickerCell extends React.PureComponent {
   props: {
     date: moment,
     title: string,
-    onChange?: (date: moment) => any,
-    _ref?: (ref: any) => any,
+    onChange: (date: moment) => any,
   }
 
   _picker: any
   openPicker = () => this._picker.onPressDate()
 
-  getRef = (ref: any) => {
-    this._picker = ref
-    this.props._ref && this.props._ref(ref)
-  }
+  getRef = (ref: any) => (this._picker = ref)
+
+  onChange = (newDate: Date) => {
+    const oldMoment = moment()
+    const changedMoment = moment(newDate)
 
-  onChange = (newDate: moment) => {
-    console.log(newDate)
-    this.props.onChange && this.props.onChange(newDate)
+    oldMoment.hours(changedMoment.hours())
+    oldMoment.minutes(changedMoment.minutes())
+
+    this.props.onChange(oldMoment)
   }
 
   render() {
-    const {date, title} = this.props
+    const format = 'h:mm A'
+
+    // We need to negate the offset, because moment inverts the offset for
+    // POSIX compatability. So, GMT-5 (CST) is shown to be GMT+5.
+    const dateInUnixMs = this.props.date.valueOf()
+    const tzOffset = -moment.tz.zone('America/Winnipeg').offset(dateInUnixMs)
+
     const accessory = (
-      <Date _ref={this.getRef} date={date.toDate()} onChange={this.onChange} />
+      <DatePicker
+        ref={this.getRef}
+        initialDate={this.props.date.toDate()}
+        minuteInterval={5}
```

Comment:
This isn't getting passed thru somewhere? The minute interval doesn't seem to be in increments of 5. Let me know if you want me to look closer at this.